api.py

```
@@ -1627,111 +1545,10 @@ async def get_all_level_roles(guild_id: str) -> list[LevelRolesGroupModel]:
-        users: list[XpBoostModel] = []
-        async for row in XpBoostModel.iter_rows(user_query, (guild_id,)):
-            users.append(row)
-
-        return {
-            "roles": roles,
-            "channels": channels,
-            "users": users,
-        }
-
+# ── XP Boost (delegated to XpBoostRepository) ──────────────────────────────────
+from repositories.xp_boost_repository import BoostTarget, XpBoostRepository, xp_boost_repo
```

Flake8 (7.3.0)

[error] 'repositories.xp_boost_repository.xp_boost_repo' imported but unused

(F401)